### DISCUSSION TITLE
雲端遊戲會是未來的趨勢嗎？

### DISCUSSION THREAD

**Discussion by ayakasanma** [1101-GS3073-C/Python-Community-GS3073-C#203] [General]
蘋果雲端遊戲市場失利 輝達、亞馬遜、微軟有望受惠

http://m.cnyes.com/news/id/4801791?utm_source=App&utm_medium=android&utm_campaign=4801791

**Comment by minniewei** [comment]
我覺得有機會耶，但也不知道會不會因此減少蘋果手機的使用者人數

**Reply by ayakasanma** [reply]
但蘋果也不是遊戲機的首選，取向可能不太一樣

**Comment by Shiba010** [comment]
其他平台該起來了，不然像現在蘋果獨大的話可能之後課金又要漲價了。

**Reply by ayakasanma** [reply]
課金漲價真的是個問題！用蘋果手機竟還要被課奢侈稅（X）手機都那麼貴了真是欺人太甚，我正在祈禱EPIC告贏蘋果XD

**Comment by jack90325** [comment]
我覺得蘋果的高額稅金真的是一個很大的問題，每次在平台上購買東西都比安卓貴

**Reply by ayakasanma** [reply]
奢侈稅好不公平Rrrrrrr
蘋果商店唯一好處就是便利商店有賣點卡...

**Comment by lx3211** [comment]
同意樓上，明明同樣的遊戲在蘋果課金硬生生比安卓貴了一些..

**Comment by BingRong1217** [comment]
感覺還蠻不錯的 多家公司互相制衡發展

**Comment by Ben6uo** [comment]
只能說是用方便來換體驗 個人感覺一般
